xmtp_mls/src/identity_updates.rs

```diff
@@ -117,7 +116,6 @@ where
             last_sequence_id,
             association_state.clone(),
         )?;
-        log::debug!("Wrote association state to cache");
```

Comment:
There's a test that relies on this log line. Even if it's not the most useful log line, probably fine to just put it back.

Reply:
added back here https://github.com/xmtp/libxmtp/pull/846/commits/f5aa568f938f0868195d0b5023e803f61c63c5bb